crates/radix-engine-toolkit-uniffi/src/builder/manifest_builder/builder.rs

```diff
@@ -1056,24 +1055,24 @@ macro_rules! manifest_args {
     ($($args: expr),*$(,)?) => {{
         use ::sbor::Encoder;
         let mut buf = ::sbor::rust::vec::Vec::new();
-        let mut encoder = radix_engine_common::data::manifest::ManifestEncoder::new(
+        let mut encoder = radix_common::data::manifest::ManifestEncoder::new(
```

Comment:
Can you please add support for the account locker methods to the builder? Examples of how we've done this in the past is at the end of the file.

Please let me know if you think that this should be a separate PR to get this in quicker for Lukasz.

Reply:
I've added all of the `AccountLocker` blueprint functions, but there is an issue with `account_locker_airdrop` in `Swift` error: `Address` does not conform to protocol `Hashable`

Reply:
I've solved this issue by making the HashMap's key a string. Will need to look into getting a Hashable address soon.